src/components/DatePicker/transform-value-to-date-or-range.ts

```diff
@@ -40,58 +35,52 @@ export const transformRelativeDateObjectToDate = (
 Read a `RelativeDate` and find the corresponding `Date` relative to the variable passed as `date` property.
 */
 export const transformRelativeDateToDate = (
-  relativeDateRange: RelativeDateRange,
+  relativeDate: RelativeDate,
   relativeAvailableVariables: VariablesBucket = [],
   variableDelimiters: VariableDelimiters = { start: '', end: '' },
 ): Date | undefined => {
   // start is always a date as relativeAvailableVariables are not customizable
-  const value = retrieveVariable(
-    relativeDateRange.date,
-    relativeAvailableVariables,
-    variableDelimiters,
-  )?.value;
+  const value = retrieveVariable(relativeDate.date, relativeAvailableVariables, variableDelimiters)
+    ?.value;
   if (!(value instanceof Date)) return;
 
-  const operator = RELATIVE_DATE_RANGE_OPERATORS[relativeDateRange.operator];
+  const operator = RELATIVE_DATE_OPERATORS[relativeDate.operator];
 
   // pass base date to UTC
   const base = DateTime.fromJSDate(value, { zone: 'UTC' }).toJSDate();
 
   // retrieve end date from luxon
   const targetDateTime = transformRelativeDateObjectToDate({
-    ...relativeDateRange,
-    quantity: Math.sign(operator.sign) * Math.abs(relativeDateRange.quantity),
+    ...relativeDate,
+    quantity: Math.sign(operator.sign) * Math.abs(relativeDate.quantity),
     date: base,
   });
 
   return DateTime.fromJSDate(targetDateTime, { zone: 'UTC' }).toJSDate();
 };
 
 /*
-Read a `RelativeDateRange` and find the corresponding `Date` relative to the variable passed as `date` property.
+Read a `RelativeDate` and find the corresponding `Date` relative to the variable passed as `date` property.
 */
-export const transformRelativeDateRangeToDateRange = (
-  relativeDateRange: RelativeDateRange,
+export const transformRelativeDateToDateRange = (
```

Comment:
Do we need to update Tucana as well? Maybe this function was used to compute the simple date selector values

Reply:
Yes indeed